content/ch-quantum-hardware/ac-Stark-shift.ipynb

```diff
@@ -46,10 +46,12 @@
     "# Importing standard Qiskit libraries and configuring account\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.    stark_tone_drive_sigma=10 * ns      #the width of the stark tone drive
</pre><p>Not width. This is Gaussian sigma of rising and falling edge </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/qiskit-community/qiskit-textbook/pull/1455/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>OK. I will fix it</p>